fedimint-client/Cargo.toml

```diff
@@ -25,7 +25,6 @@ path = "src/lib.rs"
 
 [dependencies]
 anyhow = { workspace = true }
-aquamarine = { workspace = true }
```

Comment:
This is still needed (its why your build is failing) but otherwise great cleanup!

Reply:
Done! I was running `just check` and `just check-wasm` locally but forgot to run `just build-docs`.